.changeset/ten-needles-design.md

```diff
@@ -0,0 +1,5 @@
+---
+"@neo4j/graphql": minor
+---
+
+Subscriptions stable release
```

Comment:
I would perhaps embellish this message a bit more given it's significance. Documentation can be found here, some examples can be found here... You get my drift!

Reply:
Good point, I'm not sure about how the formatting will look though

```suggestion
Subscriptions stable release. 
  * [Documentation](https://neo4j.com/docs/graphql-manual/current/subscriptions/
  * [Examples](https://github.com/neo4j/graphql/tree/dev/examples/subscriptions)
```

Reply:
I think that will format nicely - the output is markdown so it should work!